chatbots-core/src/test/groovy/io/micronaut/chatbots/core/ChatbotsConfigurationPropertiesSpec.groovy

```diff
@@ -0,0 +1,15 @@
+package io.micronaut.chatbots.core
+
+import spock.lang.Specification
+
+class ChatbotsConfigurationPropertiesSpec extends Specification {
+
+    void "ChatbotsConfiguration defaults"() {
+        given:
+        ChatbotsConfigurationProperties conf = new ChatbotsConfigurationProperties()
+
+        expect:
+        conf.enabled
+        [FileExtension.MARKDOWN, FileExtension.HTML, FileExtension.TXT] == conf.possibleStaticCommandExtensions
```

Comment:
It's "normal" to have the actual on the LHS of the comparison 🤷

Reply:
expectation on the left 

https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/Assertions.html#assertEquals-byte-byte-

Reply:
That's JUnit

https://spockframework.org/spock/docs/2.2/spock_primer.html#_expect_blocks

Expectation on the right 😜